sdk/core/core-tracing/README.md

```diff
@@ -16,40 +16,53 @@ npm install @azure/core-tracing
 
 The `@azure/core-tracing` package supports enabling tracing for Azure SDK packages, using an [OpenTelemetry](https://opentelemetry.io/) `Tracer`.
 
-By default, all libraries log with a `NoOpTracer` that takes no action.
-To change this, you have to use `setTracer` to set a new default `Tracer`.
+By default, all libraries log with a `NoOpTracer` that takes no action. To enable tracing, you will need to set a global tracer provider following the instructions in the [OpenTelemetry getting started guide](https://opentelemetry.io/docs/js/getting_started/nodejs).
 
-## Examples
+### Span Propagation
 
-### Example 1 - Setting an OpenTelemetry Tracer
+Core Tracing supports both automatic and manual span propagation. Automatic propagation is handled using OpenTelemetry's API and will work well in most scenarios.
```

Comment:
this is only true for node, right?

Reply:
Yes, good point. I remember trying this out with @opentelemetry/web and @opentelemetry/context-zone and it did work, at least for the ConsoleExporter. 

But now that I am reading a little more about context-zone and zone.js I realize it's not a great recommendation (supposedly it doesn't even work when targeting ES2017+ as per https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-context-zone-peer-dep#installation)

I'll update the README